src/e2e-test/features/gcs/source/GCSSourceAdditionalCases.feature

```diff
@@ -0,0 +1,212 @@
+@GCS_Source
+Feature: GCS source - Verification of GCS to BQ successful data transfer
+
+  @GCS_CSV_TEST @BQ_SINK_TEST @GCS_READ_RECURSIVE_TEST
+  Scenario: To verify data is getting transferred from GCS to BigQuery using read file recursive functionality
+    Given Open Datafusion Project to configure pipeline
+    When Source is GCS
+    When Sink is BigQuery
+    Then Connect source as "GCS" and sink as "BigQuery" to establish connection
+    Then Open GCS source properties
+    Then Enter GCS property projectId and reference name
+    Then Enter GCS source property path "gcsReadRecursivePath"
+    Then Toggle GCS source property skip header to true
+    Then Select GCS property format "csv"
+    Then Select GCS property read file recursive with "true"
+    Then Validate output schema with expectedSchema "gcsReadRecursivePathSchema"
+    Then Validate "GCS" plugin properties
+    Then Close the GCS properties
+    Then Open BigQuery sink properties
+    Then Enter the BigQuery sink mandatory properties
+    Then Validate "BigQuery" plugin properties
+    Then Close the BigQuery properties
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Verify the preview of pipeline is "success"
+    Then Click on preview data for BigQuery sink
+    Then Verify preview output schema matches the outputSchema captured in properties
+    Then Close the preview data
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Get count of no of records transferred to target BigQuery Table
+
+  @GCS_CSV_TEST @BQ_SINK_TEST @GCS_DATATYPE_1_TEST
```

Comment:
remove @GCS_CSV_TEST 
same comment for all scenarios... remove tags if not required for that scenario